src/main/java/frc/robot/subsystems/conveyor/commands/bits/TestColorSensor.java

```diff
@@ -0,0 +1,34 @@
+package frc.robot.subsystems.conveyor.commands.bits;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj2.command.RunCommand;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class TestColorSensor extends SequentialCommandGroup {
```

Comment:
Just an idea, maybe at the end of all the tests have a final log that shows whether the test has passed. This is in case the person testing misses the prints while the test is running